DependencyInjection/LiipImagineExtension.php

```diff
@@ -57,7 +57,7 @@ public function getConfiguration(array $config, ContainerBuilder $container)
     }
 
     /**
-     * @see Symfony\Component\DependencyInjection\Extension.ExtensionInterface::load()
+     * @see \Symfony\Component\DependencyInjection\Extension.ExtensionInterface::load()
```

Comment:
This is being migrated everywhere (slowly, they are everywhere) as many IDEs try to resolve these as within the *current namespace* when they are not declared as a root FQCN, instead of how they should be resolved, as true FQCN.